scenes/game_elements/characters/player/components/player.gd

```diff
@@ -11,6 +11,7 @@ enum Mode {
 	COZY,
 	## Player is engaged in combat. Player can use combat actions.
 	FIGHTING,
+	CAUGHT
```

Comment:
I wonder if this really has to be a new player mode. For the stealth game the player is "cozy". The only thing that's needed when caught is to play an animation, and stop handling player input.